# b_obstacle_detector:
ObstacleDetector:
ros_parameters:

hz_: 10                   # ループ周期 [Hz]
laser_step_: 5            # 何本ずつレーザを見るか(全1080本)
robot_frame_: "base_link" # frame_id

# --- はずれ値対策 ---
ignore_dist_: 0.1         # はずれ値とみなす距離 [m]